Synchronize css/cssom WPT

annevk opened the pull request:
#### 706305248d7949685acb826bab427aff851d3c92
<pre>
Synchronize css/cssom WPT
<a href="https://bugs.webkit.org/show_bug.cgi?id=257741">https://bugs.webkit.org/show_bug.cgi?id=257741</a>
rdar://110310268

Reviewed by Tim Nguyen.

Synchronize web-platform-tests/css/cssom up to upstream commit
57c5006103146974adb3af50d325c3b6ce7153d0, except for
CSSStyleSheet-constructable-concat.html which does not pass as a
reference test on CI and does pass partially in its current state.

* LayoutTests/imported/w3c/web-platform-tests/css/cssom/CSSFontFeatureValuesRule-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/CSSFontFeatureValuesRule.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/adoptedstylesheets-observablearray.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/css-stylesheet-replaceSync-null-deref-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/css-stylesheet-replaceSync-null-deref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/getComputedStyle-insets-absolute.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/getComputedStyle-insets-fixed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/getComputedStyle-insets-nobox.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/getComputedStyle-insets-relative.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/getComputedStyle-insets-static.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/getComputedStyle-insets-sticky.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/support/getComputedStyle-insets.js:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/xml-stylesheet-pi-in-doctype-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/xml-stylesheet-pi-in-doctype.xhtml: Added.

Canonical link: <a href="https://commits.webkit.org/264934@main">https://commits.webkit.org/264934@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/e6b546a262628bfba196463c9db607a53b7056a5

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/9180 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/9461 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/9689 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/10836 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/9102 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/11449 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/9428 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/11961 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/9327 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/10275 "Passed tests") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/7973 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/10992 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/7571 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/8378 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/15842 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/8673 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/8525 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/11846 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/9025 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/7364 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/8254 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/8285 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/2224 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/12477 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/8786 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Looks good, thanks!

Can you add the upstream commit revision in the commit message?

>  except for
CSSStyleSheet-constructable-concat.html which does not pass as a
reference test on CI.

We usually import everything and skip if needed.

annevk committed:
Synchronize css/cssom WPT
https://bugs.webkit.org/show_bug.cgi?id=257741
rdar://110310268

Reviewed by Tim Nguyen.

Synchronize web-platform-tests/css/cssom up to upstream commit
57c5006103146974adb3af50d325c3b6ce7153d0, except for
CSSStyleSheet-constructable-concat.html which does not pass as a
reference test on CI and does pass partially in its current state.

* LayoutTests/imported/w3c/web-platform-tests/css/cssom/CSSFontFeatureValuesRule-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/CSSFontFeatureValuesRule.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/adoptedstylesheets-observablearray.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/css-stylesheet-replaceSync-null-deref-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/css-stylesheet-replaceSync-null-deref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/getComputedStyle-insets-absolute.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/getComputedStyle-insets-fixed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/getComputedStyle-insets-nobox.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/getComputedStyle-insets-relative.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/getComputedStyle-insets-static.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/getComputedStyle-insets-sticky.html:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/support/getComputedStyle-insets.js:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/xml-stylesheet-pi-in-doctype-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/cssom/xml-stylesheet-pi-in-doctype.xhtml: Added.

Canonical link: https://commits.webkit.org/264934@main
